### PR TITLE
Alpha Normalised Cross Entropy Loss

### PR DESCRIPTION
**Pull request by cdunne97**
## Scope
- Change loss function from pytorch binary cross entropy loss to pytorch cross-entropy loss
- Add normalisation factor to normalise the cross entropy loss

## Implementation
- Changed pytorch loss function in alpha/module/alpha_trainer.py training
- Added normalisation factor in alpha/module/alpha_trainer.py initialisation

## Emoji Guide
**For reviewers: Emojis can be added to comments to call out blocking versus non-blocking feedback.**
E.g: Praise, minor suggestions, or clarifying questions that don’t block merging the PR.
> 🟢 Nice refactor!
> 🟡 Why was the default value removed?
E.g: Blocking feedback must be addressed before merging.
> 🔴 This change will break something important

| | | |
| --- | --- | --- |
| Blocking | 🔴 ❌ 🚨 | Red |
| Non-blocking | 🟡 💡 🤔 💭 | Yellow, thinking, etc |
| Praise | 🟢 💚 😍 👍 🙌 | Green, hearts, positive emojis, etc |

### CODE REVIEW

**Comment by cdunne97** [comment]
Redoing this. Some problems have been discovered.
